sound/soc/sof/ipc3-topology.c

```diff
@@ -1593,24 +1593,23 @@ static int sof_ipc3_control_load_bytes(struct snd_sof_dev *sdev, struct snd_sof_
 	struct sof_ipc_ctrl_data *cdata;
```

Comment:
Missing fixes tag:
Fixes: b5cee8feb1d48 ("ASoC: SOF: topology: Make control parsing IPC agnostic")

Reply:
@ujfalusi are you planning to update this branch?

Reply:
Oh, OK, I'll update with the Fixes tag. @ranj063, can you take a look if I got this right?